frontend/packages/shared/src/types/ComponentSpecificConfig.ts

```diff
@@ -368,7 +370,16 @@ export type ComponentSpecificConfig<T extends ComponentType = ComponentType> = {
       hideBottomBorder?: boolean;
     };
   };
-  [ComponentType.Summary2]: {};
+  [ComponentType.Summary2]: {
+    target: {
+      type?: SummaryTargetType;
+      id?: string;
+      taskId?: string;
+    };
+    showPageInAccordion?: boolean;
+    hideEmptyFields?: boolean;
+    // overrides: SummaryOverride[];
```

Comment:
Is this supposed to be commented out? 😄

Reply:
This was commented out as it is more part #13521, but it's not relevant for this task, so I removed it 👍